doc/nrf/protocols/lte/overview.rst

```diff
@@ -0,0 +1,164 @@
+|                                  | but networks use only            |                                  |
+|                                  | half-duplex FDD                  |                                  |
++----------------------------------+----------------------------------+----------------------------------+
+| Modulation in :term:`Uplink (UL)`| QPSK or 16QAM in both UL and DL  | BPSK or QPSK in UL depending on  |
+| and :term:`Downlink (DL)`        | depending on signal quality      | signal quality,                  |
+|                                  |                                  | only QPSK in DL                  |
++----------------------------------+----------------------------------+----------------------------------+
+
+Network support
+================
+
+A factor governing the selection of the access technology is the MNO support in the areas where you wish to deploy your product.
+Most cellular operators support both LTE-M and NB-IoT, but there are geographic locations where only one of these is supported.
+See `Mobile IoT deployment map`_ for more information.
+
+Certifications
+==============
+
+Some MNOs, such as Verizon and Vodafone, have their own certification requirements.
+In such cases, it is advised to engage with the MNO certification programs and contact them at the earliest.
+This is to receive certification-related requirements and better understand the potential certification costs, processes, and timelines.
+But many operators only require GCF (Global Certification Forum) or PTCRB certifications that devices work as per standard, and regulatory certifications.
+
+For more information about certification related to the nRF91 Series SiP, refer to the following pages:
+
+* `nRF91 Series certifications`_
+* `Certifying a cellular IoT device`_
+
+SIM card support
+================
+
+SIMs that are used with the nRF91 Series devices must support LTE-M, NB-IoT, or both.
+The iBasis SIM is bundled with the nRF9160 DK and Thingy:91, the Onomondo SIM with an nRF91x1 DK and Thingy:91 X, and the Wireless Logic SIM card with the nRF9151 DK and Thingy:91 X.
+Check the `iBasis IoT network coverage`_, `Onomondo LTE-M coverage`_, `Onomondo NB-IoT coverage`_, or `Wireless Logic LTE-M/NB-IoT network coverage`_ pages to see the network coverage for different countries, depending on the SIM card you are using.
+
+Software SIM
+------------
+
+The nRF91 Series supports software SIM, which allows the usage of software SIM-based solutions to reduce energy consumption associated with physical SIMs.
+The following are some of the key advantages of using software SIM:
+
+* Cost savings and simplified design by excluding the SIM hardware layer.
+* Power savings, especially in idle mode using :term:`Extended Discontinuous Reception (eDRX)`.
+
+The traditional carriers (not MVNOs like iBasis or Onomondo) do not allow their profiles to be provisioned or uploaded into SoftSIMs.
```

Comment:
not neccesarly the case anymore, suggest to remove section

Reply:
Removed the paragraph but kept the link to the Softsim section on the product page - https://www.nordicsemi.com/Products/Low-power-cellular-IoT/Unique-Features?lang=en#infotabs.